charts/pomerium/templates/proxy-deployment.yaml

```diff
@@ -79,6 +79,11 @@ spec:
             secretKeyRef:
               name: {{ $secretName }}
               key: shared-secret
+        - name: SIGNING_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "pomerium.proxy.jwtSecret.name" . }}
+              key: signing.key
```

Comment:
@tgomas why use `.` as a separator, other secrets seem to use a `-`.

Reply:
actually, it's not so obvious, see the certificates secret that use a `.`, @travisgroth an opinion?

Reply:
because TLS secrets use a `.` (e.g. `tls.key` or `ca.crt`)

Reply:
now that it's in an Opaque secret, a `-` is used as separator